java/runtime/src/main/java/org/ray/runtime/util/NetworkUtil.java

```diff
@@ -49,16 +52,13 @@ public static String getIpAddress(String interfaceName) {
   }
 
   public static int getUnusedPort() {
-    int port;
-    try {
-      ServerSocket ss = new ServerSocket();
-      ss.bind(new InetSocketAddress(0));
-      port = ss.getLocalPort();
-      ss.close();
-    } catch (Exception e) {
-      throw new RuntimeException("Failed to bind to an available port.", e);
+    Random random = new Random();
```

Comment:
The `Random` instance should be static in case of calling `getUnusedPort` multiple times in a short time. And ever better, use [ThreadLocalRandom](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ThreadLocalRandom.html).